scripts/linters/pylint_extensions.py

```diff
@@ -2017,8 +2037,35 @@ def _process_module_tokens(self, tokens, node):
 
                     comment_line_number = line_num
 
-                if re.search(r'(\s*type:\s*ignore)', line):
-                    if 'type: ignore[no-untyped-call]' in line:
+                if re.search(r'(\s*type:\s*ignore\[)', line):
+                    prohibited_error_code_present = False
+
+                    error_codes = re.search(
+                        r'(\s*type:\s*ignore)\[([a-z-\s\,]*)\]', line
+                    ).group(2)
+
+                    encountered_error_codes = []
+                    encountered_prohibited_error_codes = []
+                    for error_code in error_codes.split(','):
+                        error_code = error_code.strip()
+                        if (
+                            error_code not in
+                            self.config.allowed_type_ignore_error_codes
+                        ):
+                            prohibited_error_code_present = True
+                            encountered_prohibited_error_codes.append(
+                                error_code
+                            )
+                        encountered_error_codes.append(error_code)
+
+                    if prohibited_error_code_present:
```

Comment:
You can put `encountered_prohibited_error_codes` here instead. It will evaluate as being truthy if the list isn't empty, and then you don't need the `prohibited_error_code_present` variable

Reply:
Done!